vespa-athenz/src/test/java/com/yahoo/vespa/athenz/identityprovider/client/IdentityDocumentSignerTest.java

```diff
@@ -60,10 +61,10 @@ void ignores_cluster_type() {
 
         var docWithoutClusterType = new SignedIdentityDocument(
                 signature, KEY_VERSION, id, providerService, DEFAULT_DOCUMENT_VERSION, configserverHostname,
-                instanceHostname, createdAt, ipAddresses, identityType, null);
+                instanceHostname, createdAt, ipAddresses, identityType, null, ztsUrl);
```

Comment:
Consider passing `null` instead of `ztsUrl` and rename test to `ignores_cluster_type_and_zts_url`